src/encoded/types/antibody_lot.py

```diff
@@ -165,9 +165,10 @@ def lot_reviews(characterizations, targets, request):
         if 'histone' in target['investigated_as']:
             is_histone = True
 
-        organism = target['organism']
+        organism = target.get('organism')
         target_organisms = {'all': []}
-        target_organisms['all'].append(organism)
+        if organism:
+            target_organisms['all'].append(organism)
```

Comment:
Why are we even using `all`?

Reply:
As mentioned above, it doesn't make sense and will be addressed in ENCD-4299.